assets/js/src/form_editor/components/size_settings.tsx

```diff
@@ -18,7 +18,7 @@ type Props = {
   onChange: (value: SizeDefinition) => void;
 }
 
-export const SizeSettings = ({
+export const SizeSettings: React.FunctionComponent<Props> = ({
```

Comment:
I guess the difference is that in case of assigning to a variable I used this `const SizeSettings: React.FunctionComponent<Props>`. But if there is no variable I had to specify what is actually returned from the component and it is `JSX.Element` both are possible and because I didn't want to introduce a new variable I just used this. I guess you can pick whichever you want.

Reply:
Oh, I see! The function itself is of type `React.FunctionComponent` but it returns JSX.Element. Thanks!